packages/app/src/outbox/services/message-entry-handler.service.ts

```diff
@@ -4,22 +4,24 @@
  */
 
 import { MessageRepository } from "App/messages/repositories"
-import { OutboxEntry, OutboxEntryChange } from "@mudita/pure"
+import { OutboxEntry, OutboxEntryChange, OutboxEntryType } from "@mudita/pure"
 import { RequestResponseStatus } from "App/core/types/request-response.interface"
 import { EntryHandler } from "App/outbox/services/entry-handler.type"
 import { MessageService } from "App/messages/services"
+import { ThreadEntryHandlerService } from "App/outbox/services/thread-entry-handler.service"
 
 export class MessageEntryHandlerService implements EntryHandler {
   constructor(
     public messageService: MessageService,
     private messageRepository: MessageRepository,
+    private threadEntryHandlerService: ThreadEntryHandlerService
   ) {}
 
-  public async handleEntry(entry: OutboxEntry): Promise<void> {
+  public handleEntry = async (entry: OutboxEntry): Promise<void> => {
```

Comment:
why it's a property instead of a method?

Reply:
because the class is binding to object map structure, example [link](https://github.com/mudita/mudita-center/pull/699/files/7046a6ac12641366ba50c8cf6818f54c5752c5ac#diff-f1ee363e581251211cf6b57c3cffebaf3c5c51a9af9a4b7ff8a0a23905459aa7R56)

Reply:
no, because the context is lost. The binding isn't work too because the binding context is another place.